libs/interfaces/src/lib/design/design-dimension.enum.ts

```diff
@@ -3,4 +3,5 @@ export enum DesignDimensionEnum {
   medium = 'MEDIUM',
   large = 'LARGE',
   oneSize = 'ONE_SIZE',
+  mini = 'MINI',
```

Comment:
We shouldn't add this as an extra type. There is nothing that uses it at the moment, and it would need to be reflected in the price calculation services etc